regression/apps/mongoose/main.c

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2020 Cesanta Software Limited
+// All rights reserved
```

Comment:
Does this file have a license and whether is its license compatible with that of Asterinas? Asterinas requires that any file has a valid SPDX license header.

Reply:
Received! I have rewritten an example to keep the license consistent with Asterinas.